hazelcast/src/test/java/com/hazelcast/cache/CacheBasicAbstractTest.java

```diff
@@ -1044,6 +1045,26 @@ public void cacheManagerOfCache_cannotBeOverwrittenConcurrently() throws Excepti
         assertEquals(1, illegalStateExceptionCount.get());
     }
 
+    @Test
+    public void testUpdatingExpiryPolicyOfCacheEntry() {
+        ICache<Integer, String> cache = createCache();
+        cache.put(1, "value", new CreatedExpiryPolicy(new Duration(TimeUnit.MILLISECONDS, 5000)));
+
+        assertTrue(cache.setExpiryPolicy(1, new CreatedExpiryPolicy(new Duration(TimeUnit.MILLISECONDS, 6000))));
+        assertTrueEventually("expected the key to be expired", () ->
+                Assertions.assertNull(cache.get(1)), 10);
```

Comment:
Nit-pick but you're using the jupiter extension `Assertions` in a legacy JUnit test class - I'd prefer to see `Assert` used here

Reply:
> Nit-pick but you're using the jupiter extension `Assertions` in a legacy JUnit test class - I'd prefer to see `Assert` used here

Good spot - I don't think you can mix&match like that.